### PR TITLE
Some Repaired LV changes and makes a Repaired New Varadero variant

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
# About the pull request

Replaces the normal fax machine in the Corpo Dome with a CL one.
Unlocks the Corpo Dome storage room.
Fixes some fences issues from my repairs LV PR near the Comms Dome.

Removes xeno/blood/broken/corpses and alot of things that would ruin a poor little UA Colony like NV. Keeps the predship but adds more labcoat stuff there just before the 'outbreak' when they were researching it.

# Explain why it's good for the game

Kirsh uses NV for colony fall alot, so I did an LV and pretty'd it up, if anyone has any recommendations or changes they want please let me know.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
mapadd: added repaired New Varadero
maptweak: tweaked repaired LV
/:cl:

### CODE REVIEW

File: code/__DEFINES/__game.dm [hunk 1]
```diff
@@ -26,8 +26,8 @@ block( \
 
 
 #define MAP_ICE_COLONY "Ice Colony" // Highpop only
-#define MAP_LV_624 "LV-624"
-#define MAP_LV_624_REPAIRED "LV-624 Repaired"
+#define MAP_LV_624 "LV-624 Repaired"
```
Comment (FslashN) [inline code/__DEFINES/__game.dm:29]: I'm confused here. `MAP_LV_624 "LV-624 Repaired"` shouldn't this be `MAP_LV_624_REPAIRED "LV-624 Repaired"`?

File: code/__DEFINES/__game.dm [hunk 2]
```diff
@@ -26,8 +26,8 @@ block( \
...
+#define MAP_LV_624_REPAIRED "LV-624"
```
Comment (FslashN) [inline code/__DEFINES/__game.dm:30]: Are these mixed up?

**Review by FslashN — CHANGES_REQUESTED** [review]
A little question here, but otherwise seems fine.

File: code/__DEFINES/__game.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/__DEFINES/__game.dm]: Fixed, was an accident.

File: code/__DEFINES/__game.dm (hunk 2 above)
Comment (BadAtThisGame302) [reply code/__DEFINES/__game.dm]: Fixed, was an accident.

**Review by FslashN — APPROVED** [review]

**Comment by FslashN** [comment]
Should be fine to merge as it's just an additional map for GMs to use.

**Pull request merged**